pkg/odo/cli/pipelines/init.go

```diff
@@ -92,9 +92,9 @@ func NewCmdInit(name, fullName string) *cobra.Command {
 		},
 	}
 
-	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "GitOps repository e.g. https://github.com/organisation/repository")
+	initCmd.Flags().StringVar(&o.GitOpsRepoURL, "gitops-repo-url", "", "provide a GitOps repository(it could be GitHub or GitLab) e.g. https://github.com/organisation/repository")
```

Comment:
I don't think "(it could be GitHub or GitLab)" sound good.   What if we support another Git provider?   Are we going to add to this list?   I think we can leave that out.   Something like below is fine.
"provide GitOps repository URL (e.g. ....)"

Reply:
made the changes.